main.go

```diff
@@ -425,8 +469,18 @@ func recycleIPTables(nw ip.IP4Net, lease *subnet.Lease) error {
 }
 
 func recycleIP6Tables(nw ip.IP6Net, lease *subnet.Lease) error {
-	prevNetwork := ReadIP6CIDRFromSubnetFile(opts.subnetFile, "FLANNEL_IPV6_NETWORK")
-	prevSubnet := ReadIP6CIDRFromSubnetFile(opts.subnetFile, "FLANNEL_IPV6_SUBNET")
+	prevNetworks := ReadIP6CIDRsFromSubnetFile(opts.subnetFile, "FLANNEL_IPV6_NETWORK")
+	prevSubnet := ReadIP6CIDRFromSubnetFile(opts.subnetFile, "FLANNEL_IPV6_NETWORK")
```

Comment:
Why are you removing SUBNET here in case of IPv6? NETWORK should be the lists of networks of the cluster and SUBNET is the network of the node.

Reply:
you're right, that's a mistake